apps/sms/style/root.css

```diff
@@ -177,3 +177,11 @@ menu, nav, section, video, audio, canvas, progress, meter, time {
   margin: 0;
 }
 
+/* Dialogs */
+body.dialog-animating {
+  pointer-events: none;
+}
+
+body.dialog-animating .disable-when-dialog-animates {
+  pointer-events: none;
+}
```

Comment:
Do we prefer repeat this style whenever we need it, or it has more sense to make it accessible, from the shared style folder?